.github/workflows/version-code-and-release.yml

```
@@ -11,12 +11,7 @@ on:
       - 'detection_rules/**'
       - 'tests/**'
       - '**/*.md'
-    types: [opened, reopened, synchronize, labeled]
-  push:
-    branches:
-      - main
-    paths:
-      - 'pyproject.toml'
+    types: [opened, reopened, synchronize, labeled, closed]
```

```suggestion
    types: [opened, reopened, synchronize, labeled]
```

Just double checking, are we sure we want closed?

---

Oh wait, we do want closed because we would want this check to run after a merge too right?

---

yes thats for the last step.